### PR TITLE
fix(Heading/react): remove default heading margin

### PR DESCRIPTION
**Pull request by krishnaacc123**

### CODE REVIEW

File: src/atoms/Heading/Heading.web.js [hunk 1]
```diff
@@ -52,11 +52,17 @@ const headingLevel = {
   medium: 'h5',
 };
 
+const StyledText = styled(Text)`
```
Comment (dhruvdutt) [inline src/atoms/Heading/Heading.web.js:55]: Use `Space` component.

Comment (krishnaacc123) [reply src/atoms/Heading/Heading.web.js]: added

File: src/atoms/Heading/Heading.web.js [hunk 2]
```diff
@@ -56,18 +57,20 @@ const Heading = ({ size, color, children, maxLines, weight }) => {
...
-      color={color}
-      maxLines={maxLines}
-      _weight={weight}
-      _lineHeight={styles.lineHeight({ theme, size, weight })}
-      _letterSpacing="small"
-      {...automation('ds-heading')}
-    >
-      {children}
-    </Text>
+    <Space margin={[0]}>
```
Comment (kamleshchandnani) [inline src/atoms/Heading/Heading.web.js:60]:
```suggestion
    <Space margin={0}>
```

Else it'll just apply apply margin-top:0 with array.

Comment (dhruvdutt) [reply src/atoms/Heading/Heading.web.js:60]: Wut? 😮 Didn't know we have done this implemention.

Comment (kamleshchandnani) [reply src/atoms/Heading/Heading.web.js:60]: No my bad.

**Pull request merged**